multi_imbalance/resampling/spider.py

```diff
@@ -209,14 +221,16 @@ def _min_cost_classes(self, x, DS):
         for cj in C:
             s = 0
             for ci in C:
-                s += ((kneighbors[:, -1] == ci).astype(int).sum() / self.k) * self.cost[C.index(ci), C.index(cj)]
+                s += ((kneighbors[:, -1] == ci).astype(int).sum() / self.k) * self.cost[
+                    C.index(ci), C.index(cj)
+                ]
             vals.append(s)
         C = np.array(C)
         vals = np.array(vals)
         vals = np.round(vals, 6)
         return C[vals == vals[np.argmin(vals)]]
 
-    def _relabel_nn(self, x):
+    def _relabel_nn(self, x: np.ndarray):
```

Comment:
```suggestion
    def _relabel_nn(self, x: np.ndarray) -> None:
```
[nit] You could add None if method does not return anything

Reply:
Done